torch/csrc/jit/passes/alias_analysis.cpp

```diff
@@ -371,6 +371,7 @@ void AliasDb::analyzeImpl(Node* node) {
       return;
     case prim::CallFunction:
     case prim::CallMethod:
+    case prim::ToList:
```

Comment:
Since toList doesn't mutate its input and produces a fresh alias, we can put toList with the other `analyzeCreator` cases. analyzeConservative assumes that all inputs and outputs are written to.

Reply:
Done.